manifests/m/MariaDB/Server/10.6.9.0/MariaDB.Server.locale.en-US.yaml

```
@@ -0,0 +1,25 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: MariaDB.Server
+PackageVersion: 10.6.9.0
+PackageLocale: en-US
+Publisher: MariaDB Corporation Ab
+PublisherUrl: https://mariadb.org
+PrivacyUrl: https://mariadb.org/privacy-policy/
+Author: MariaDB Corporation Ab
+PackageName: MariaDB 10.6
```

```suggestion
PackageName: MariaDB
```
since it's already in `AppsAndFeaturesEntries`